src/main/java/com/google/impactdashboard/api_utilities/RecommendationRetriever.java

```diff
@@ -0,0 +1,31 @@
+package com.google.impactdashboard.api_utilities;
+
+import com.google.cloud.recommender.v1beta1.ListRecommendationsRequest;
```

Comment:
I thought recommender API is already in GA. Any reason of using v1beta1?

Reply:
There is not reason currently. The only reason I chose v1beta1 over v1 is the possibility of using the insight classes provided in beta1 in the future.

Reply:
I'll change this to v1 until we know if we need v1beta1